cmd/make-bm-worker/main.go

```diff
@@ -81,11 +84,16 @@ func main() {
 		os.Exit(1)
 	}
 
+	if *hardwareProfile == "" {
+		*hardwareProfile = "libvirt"
```

Comment:
Let’s leave it empty if not specified and use the default

Reply:
Fixed!